pkg/controller/apiclient/mem/instance.go

```diff
@@ -36,6 +37,11 @@ func newInstanceClient() *instanceClient {
 	return &instanceClient{instances: make(map[string]*servicecatalog.Instance)}
 }
 
+// NewPopulatedInstanceClient returns a pre-populated in-memory instance client implementation
+func NewPopulatedInstanceClient(inst map[string]*servicecatalog.Instance) apiclient.InstanceClient {
+	return &instanceClient{instances: inst}
+}
+
```

Comment:
Another non-blocking nit: I would consider `instances` more readable than `inst`. My objection isn't to the shortening of the word, but rather to the fact that `inst` sounds singular.